boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -15,6 +15,9 @@ Bluetooth™, GPS, displays, and camera sensors. As with other i.MX processors,
 i.MX RT1050 also has rich audio and video features, including LCD display,
 basic 2D graphics, camera interface, SPDIF, and I2S audio interface.
 
+The following document refers to the discontinued MIMXRT1050-EVK board, for the
+MIMXRT1050-EVKB board, refer to `Board Revisions`_ section.
```

Comment:
I think you need to add a link target in the board revisions section for this link to work. @dbkinder?

Reply:
This link refers to the (implicit) target for the section title "Board Revisions" found later in this document.  (If you were referring to a section title in another document, you'd need to add an explicit target.)  So, this reference is OK as is.

Reply:
Ok, thanks